src/get-started/editor.md

```diff
@@ -41,92 +50,132 @@ that's OK, skip ahead to the [next step: Test drive][].
 
 ## Install VS Code
 
-VS Code is a lightweight editor with complete Flutter
-app execution and debug support.
+[VS Code][] is a code editor to build and debug apps.
+With the Flutter plugin installed, you can compile, deploy, and debug
+Flutter apps.
+
+To install the latest version of VS Code,
+follow Microsoft's instructions for the relevant platform:
+
+- [Install on macOS][]
+- [Install on Windows][]
+- [Install on Linux][]
+
+[VS Code]: https://code.visualstudio.com/
+[Install on macOS]: https://code.visualstudio.com/docs/setup/mac
+[Install on Windows]: https://code.visualstudio.com/docs/setup/windows
+[Install on Linux]: https://code.visualstudio.com/docs/setup/linux
+
+## Install the VS Code Flutter plugin
```

Comment:
Consider additionally documenting the alternative of clicking "Install" from the https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter page which should open the editor for you and complete some (or all?) of these steps. It seems like an easier experience for users.

Reply:
**comment:** I like that. I'll check it out.